src/native/external/llvm-libunwind/src/DwarfInstructions.hpp

```diff
@@ -262,13 +269,13 @@ int DwarfInstructions<A, R>::stepWithDwarf(A &addressSpace, pint_t pc,
       //
       // We set the SP here to the CFA, allowing for it to be overridden
       // by a CFI directive later on.
-      newRegisters.setSP(cfa);
+      newRegisters.setSP(cfa, 0);
 
       pint_t returnAddress = 0;
-      constexpr int lastReg = R::lastDwarfRegNum();
-      static_assert(static_cast<int>(CFI_Parser<A>::kMaxRegisterNumber) >=
-                        lastReg,
-                    "register range too large");
+      pint_t returnAddressLocation = 0;
+      const int lastReg = R::lastDwarfRegNum();
```

Comment:
This looks like incorrectly resolved conflict. Should this be `constexpr` and `static_assert` from upstream?

Reply:
These are from the first commit. After the second commit, these are no longer changed in the final diff.